src/contract/compiler/Cli.ts

```diff
@@ -17,14 +17,19 @@ const getPackagePath = (): string => {
 };
 
 /**
- * A wrapper around aesophia_cli, available only in Node.js
- * Assumes that `escript` is available in PATH.
+ * A wrapper around aesophia_cli, available only in Node.js.
+ * Requires Erlang installed, assumes that `escript` is available in PATH.
  */
 export default class CompilerCli extends CompilerBase {
   #path: string;
 
   #ensureCompatibleVersion = Promise.resolve();
 
+  /**
+   * @param compilerPath - A path to aesophia_cli binary, by default uses the integrated one
```

Comment:
"integrated one" means that the CLI is actually shipped along with the SDK. do I understand correctly?

Reply:
I assume that windows is not supported here using the CLI compiler, right? 🤔

Reply:
that's cool. wouldn't have expected the CLI binary of the compiler to work with windows

Reply:
it is an Erlang VM binary